### PR TITLE
Hot fix/fix deploy

### PR DESCRIPTION
**Pull request by planktonic**
The deploy-to-github-pages action that happens on merging to `main` fails when building storybook. Debugging here

### CODE REVIEW

**Comment by planktonic** [comment]
This now works, (take a look at the actions tab for the repo, the last run was green), but I’ve limited the deploy to only run when we push certain changes to the main branch, so there is no newer deploy entry there (until this PR gets merged)

File: yarn.lock [hunk 1]
```diff
@@ -2749,13 +2749,6 @@ async@0.9.x:
   resolved "https://registry.yarnpkg.com/async/-/async-0.9.2.tgz#aea74d5e61c1f899613bf64bda66d4c78f2fd17d"
   integrity sha1-rqdNXmHB+JlhO/ZL2mbUx48v0X0=
 
-async@^2.6.1:
-  version "2.6.3"
-  resolved "https://registry.yarnpkg.com/async/-/async-2.6.3.tgz#d72625e2344a3656e3a3ad4fa749fa83299d82ff"
-  integrity sha512-zflvls11DCy+dQWzTW2dzuilv8Z5X/pjfmZOWba6TNIVDm+2UDaJmXSOXlasHKfNBs8oo3M0aT50fDEWfKZjXg==
-  dependencies:
-    lodash "^4.17.14"
-
```
Comment (nickgnd) [inline yarn.lock:2758]: Why all these changes in the `yarn.lock`? You didn't update any dependencies.

**Review by nickgnd — APPROVED** [review]
I don't have a good knowledge of github actions, but changes to the pipeline look good.
I'm wondering why so many changes in the `yarn.lock` file, did you update any deps?

**Review by klappradla — APPROVED** [review]
Changes in the actions look good ✨ 

Changes to the JS dependencies seem to come from another PR 👀

**Comment by planktonic** [comment]
@nickgnd @klappradla thanks! Yes, I mistakenly included those yarn.lock changes in #390 — that PR originally used a `gh-pages` package to publish to GitHub Pages, but when I switched to using GitHub Actions (which required no extra packages) in that PR, I forgot to update the yarn.lock there :/

I could separate those changes into a new PR?

**Comment by klappradla** [comment]
> in that PR, I forgot to update the yarn.lock there :/

Fine to have them in this one then 🤷

**Pull request merged**